examples/kind/kind.values.yaml

```diff
@@ -0,0 +1,27 @@
+# Values for a small local test deployment using Kind
```

Comment:
I need to add some more examples, but will do that in a follow-up to keep this PR small